packages/react/src/components/TimePicker/TimePicker.types.ts

```diff
@@ -0,0 +1,41 @@
+import { IComboBoxProps } from '@fluentui/react/lib/ComboBox';
```

Comment:
similarly here, this import breaks the build

Reply:
FIxed